### DISCUSSION TITLE
Android Version???

### DISCUSSION THREAD

**Discussion by mahdi137921** [Syncplay/syncplay#481] [Archive]
Hey There...
Is it possible for you to make this app works on android?
That would be great

**Comment by daniel-123** [comment]
Creating android client for Syncplay is surprisingly complicated for two main reasons:

- Due to how Android works it has to effectively be a video player that *also* is able to connect to Syncplay.
- None of usual developers have any experience with writing or maintaining Android apps.

Because of that there are currently no plans to make an official Android app.

That said an unofficial client does exist: https://github.com/mo3rfan/syncplayer

**Comment by soredake** [comment]
Check this https://github.com/chromaticnoob/syncplay-android/

**Comment by Et0h** [comment]
This discussion is being locked as there are no plans for the main Syncplay team to make their own Android client and the third-party Android implementations are now listed at #570.
